pkg/ipamd/ipamd.go

```diff
@@ -292,6 +300,19 @@ func prometheusRegister() {
 	}
 }
 
+// containsInsufficientCidrBlocksError returns whether exceeds ENI's IP address limit
+func containsInsufficientCidrBlocksError(err error) bool {
```

Comment:
let's use below, this way works even if err is a wrapped error.
```
var awsErr awserr.Error
if errors.As(err, &awsErr) {
	return awsErr.Code() == "InsufficientCidrBlocks"
}
```

Reply:
sure 👍